firstvoices/backend/views/import_job_views.py

```diff
@@ -29,7 +32,13 @@
         parameters=[site_slug_parameter],
     ),
     retrieve=extend_schema(
-        description=_("Details about a specific batch import job."),
+        description=_(
+            "Details about a specific batch import job. "
+            "Includes standard task fields: status, task ID, and error message. "
+            "If doing a dry-run, validationTaskId, validationStatus, and validationReport are also added. "
```

Comment:
I think "dry-run" is an internal concept that we don't need to mention in the docs, and the specific fields are listed in the docs automatically. This could just be "Details about a specific batch import job."